WordPress/Classes/Networking/WordPressComServiceRemote.m

```diff
@@ -38,31 +38,12 @@ - (void)createWPComAccountWithEmail:(NSString *)email
     NSParameterAssert([username isKindOfClass:[NSString class]]);
     NSParameterAssert([password isKindOfClass:[NSString class]]);
     
-    void (^successBlock)(AFHTTPRequestOperation *, id) = ^(AFHTTPRequestOperation *operation, id responseObject) {
+    void (^successBlock)(id responseObject, NSHTTPURLResponse *httpResponse) = ^(id responseObject, NSHTTPURLResponse *httpResponse) {
```

Comment:
You don't need to name parameters on the left side of the assignment as it's just a type annotation.